addOns/ascanrulesAlpha/src/main/resources/org/zaproxy/zap/extension/ascanrulesAlpha/resources/LdapErrors.properties

```diff
@@ -10,7 +10,7 @@ ascanalpha.ldapinjection.activedirectory.errormessages=0x00005011L:0x00005012L:0
g this base class. They have no target entry unlike the rest of the SingleReplyRequests:Unidentified single reply request/response type:Unexpected base scope.:TimeUnit not supported:Unsupported protocol type:Failed to add the extended operation of type:Failed to add the SASL mechanism with the name:Failed to add NTLM provider:Empty ID:Invalid replica ID:The following schema directory from the installation layout could not be found:Unknown SchemaObject type:Could not find entry at:Cannot delete entry:not in value range:Bind requests only tunnel down into partitions if there are no authenticators to handle the mechanism:This cursor is empty and cannot return elements!:Cannot efficiently search the DIB w/o an index on attribute:Cannot access element if positioned before first.:Cannot access element if positioned after last.:AndCursors are not ordered and do not support positioning by element.:Cursor has not been positioned yet.:Could not find matchingRule to use for EqualityNode evaluation:Unrecognized leaf node:Unrecognized branch node type:Unrecognized search scope value for filter scope node:EqualityCursors only support positioning by element when a user index exists on the asserted attribute.:Could not find matchingRule to use for GreaterEqNode evaluation:LessEqCursors only support positioning by element when a user index exists on the asserted attribute.:chingRule to use for LessEqNode evaluation:NotCursors are not ordered and do not support positioning by element.:Scope Cursors are not ordered and do not support positioning by element.:ScopeNode is not of onelevel scope.:This is too inefficient without getId() on ServerEntry:OrCursors are not ordered and do not support positioning by element.:Must have 2 or more sub-expression Cursors for a disjunction:PresenceCursors do not support positioning by element without a user index on the presence attribute.:SubstringCursors may not be ordered and do not support positioning by element.:Can't delete file:ScopeNode is not of subtree scope.:There can only be 2 columns at index 0 and at 1:You didn't correctly set col names:Unrecognized mode.:Unrecognized FilterDialog command:Cannot process a Request while binding:Principal is not a valid Name:Cannot encode the KrbError object, the PDU size is:Cannot encode the KrbSafeBody object, the PDU size is:Cannot encode the KrbSafe object, the PDU size is:Cannot encode the EncKrbPrivPart object, the PDU size is:Cannot encode the KrbPriv object, the PDU size is:Cannot encode the KrbCredInfo object, the PDU size is:Cannot encode the EncKrbCredPart object, the PDU size is:Cannot encode the KrbCred object, the PDU size is:Cannot encode the EncTicketPart object, the PDU size is:Cannot encode the TypedData object, the PDU size is:The PDU length is null, this is not allowed:is not a valid KerberosString:Not a valid log file number:Not a valid log file offset:Invalid log file bufferSize/ max size is sepcified bufferSize:Log Scanner is already closed:Log content is invalid
```

Comment:
This should be reverted the error message is still the same: https://github.com/apache/directory-server/blob/4941c2565d374632193d93920e0bc870fe3a1e24/i18n/src/main/resources/org/apache/directory/server/i18n/errors.properties#L698